package.json

```diff
@@ -11,24 +11,24 @@
   },
   "description": "GitHub Action to Install the Octopus CLI",
   "devDependencies": {
-    "@types/jest": "^29.2.3",
-    "@types/node": "^18.11.9",
+    "@types/jest": "^29.2.4",
+    "@types/node": "^18.11.13",
     "@types/semver": "^7.3.13",
     "@types/tmp": "^0.2.3",
-    "@types/uuid": "^8.3.4",
-    "@typescript-eslint/eslint-plugin": "^5.43.0",
-    "@typescript-eslint/parser": "^5.43.0",
-    "@vercel/ncc": "^0.34.0",
-    "eslint": "^8.27.0",
-    "eslint-plugin-jest": "^27.1.5",
+    "@types/uuid": "^9.0.0",
+    "@typescript-eslint/eslint-plugin": "^5.46.1",
+    "@typescript-eslint/parser": "^5.46.1",
+    "@vercel/ncc": "^0.36.0",
+    "eslint": "^8.29.0",
+    "eslint-plugin-jest": "^27.1.6",
     "jest": "^29.3.1",
     "jest-circus": "^29.3.1",
-    "jest-junit": "^14.0.1",
+    "jest-junit": "^15.0.0",
```

Comment:
I see a major version bump here; these have broken us in the past... but all the tests ran and confirm it's ok, so approved